scripts/Install-VSCode.ps1

```diff
@@ -100,22 +130,50 @@ param(
 )
 
 if (!($IsLinux -or $IsOSX)) {
-
-    $codeCmdPath = "C:\Program Files (x86)\Microsoft VS Code\bin\code.cmd"
-
+    switch ($Architecture) {
+        "64-bit" {
+            if ((Get-WmiObject -Class Win32_OperatingSystem).OSArchitecture -eq "64-bit") {
```

Comment:
I think you forgot the change to use `Get-CimInstance`.  :-)  Other than that, I think this is ready to merge.

Reply:
shit! good catch. I put it in my clipboard and forgot to paste it 💀